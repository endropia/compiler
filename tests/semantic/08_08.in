procedure p(var a: integer);
	begin
	end;
begin
	p(10);
end.