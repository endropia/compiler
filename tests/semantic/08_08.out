program : Unnamed program
   procedure:
      p
      parameters: 
         type: integer
         var
         a
      stmts:
         empty   stmts:
      call
         p
            10

scope     name                          class               
------------------------------------------------------------
0         integer                       primitive type      
0         double                        primitive type      
0         boolean                       primitive type      
0         char                          primitive type      
0         string                        primitive type      
1         p                             procedure           
2         a                             parameter variable reference
